crates/oxc_wasm/src/lib.rs

```diff
@@ -234,7 +236,7 @@ impl Oxc {
                 &path,
                 source_type,
                 source_text,
-                semantic.trivias(),
+                Rc::clone(&trivias),
```

Comment:
I think this can just be `trivias`. No need to clone the `Rc`, as it's not used again after this.

Reply:
Thanks for catching that